StripeCryptoOnramp/StripeCryptoOnramp/Source/Internal/API Bindings/STPAPIClient+CryptoOnramp.swift

```diff
@@ -41,10 +43,6 @@ extension STPAPIClient {
 
         try validateSessionState(using: linkAccountInfo)
```

Comment:
```swift
private func validateSessionState(using linkAccountInfo: PaymentSheetLinkAccountInfoProtocol) throws {
    guard case .verified = linkAccountInfo.sessionState else {
        throw CryptoOnrampAPIError.linkAccountNotVerified
    }
}
```
I noticed this while making the change here, that the code below was redundant, and likely left over from a deduplication refactor when creating `validateSessionState(using:)`.